src/main/java/com/wiley/autotest/selenium/elements/upgrade/OurWebElement.java

```diff
@@ -214,28 +204,6 @@ private void clickForIgnoredScroll(WebDriverException ignoredOrNeedToScroll) {
         increment();
         //For Android error text is different and does not have any information related to clickable issue
         String ignoredOrNeedToScrollMessage = ignoredOrNeedToScroll.getMessage();
-        if (isAndroid() || ignoredOrNeedToScrollMessage.contains("is not clickable at point")) {
```

Comment:
@LaVa1804 im concerned by removing this IFs body. I realise that it might not be necessary already but lets do the following
1. remove if(isAndroid()) parts but keep the if and it's inner part for second condition (Regarding error message). 
2. add a todo saying "this code is probably non needed anymore, consider deleting it"

What do you think @LaVa1804 @estuold?